cypress/e2e/spec.cy.ts

```diff
@@ -1,5 +0,0 @@
-describe('empty spec', () => {
-  it('passes', () => {
-    cy.visit('https://example.cypress.io')
-  })
-})
```

Comment:
Where is your first test for [www.saucedemo.com](https://www.saucedemo.com/ )? How we can clarify that our framework is configured correctly and works fine using Git Hub Actions?